src/util.js

```diff
@@ -180,62 +18,6 @@ util.forEach = function(obj, f, thisArg) {
   }
 };
 
-util.reduce = function(obj, f, init, thisArg) {
-  if (obj.reduce) {
-    return obj.reduce.call(thisArg, f, init);
-  } else {
-    for (var k in obj) {
-      init = f.call(thisArg, init, obj[k], k, obj);
-    }
-    return init;
-  }
-};
-
-util.map = function(obj, f, thisArg) {
-  if (obj.map) {
-    return obj.map.call(thisArg, f);
-  } else {
-    var output = [];
-    for (var k in obj) {
-      output.push( f.call(thisArg, obj[k], k, obj));
-    }
-  }
-};
-
-util.any = function(arr, f) {
```

Comment:
Hmm, I guess, we can either move it into datalib or compass, since it was unused in vl.